src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -49,10 +49,13 @@ protected function resetService($name)
         }
         $manager->setProxyInitializer(\Closure::bind(
             function (&$wrappedInstance, LazyLoadingInterface $manager) use ($name) {
+                if (isset($this->normalizedIds[$normalizedId = strtolower($name)])) { // BC with DI v3.4
+                    $name = $this->normalizedIds[$normalizedId];
+                }
                 if (isset($this->aliases[$name])) {
                     $name = $this->aliases[$name];
                 }
-                $method = $this->methodMap[$name] ?? 'get'.$name.'Service';
+                $method = $this->methodMap[$name] ?? 'get'.strtr($name, $this->underscoreMap).'Service'; // BC with DI v3.4
```

Comment:
on 4.0, `$this->methodMap[$name]` must be set, which means the "else" path i pre-3.4, where `$this->underscoreMap` always exists.